browser.js

```diff
@@ -1,10 +1,42 @@
 var elliptic = require('elliptic');
 var BN = require('bn.js');
 module.exports = ECDH;
-
-function ECDH(curve, crypto) {
-	elliptic.rand = crypto.randomBytes;
-	this.curve = new elliptic.ec(curve);
+var aliases = {
+	secp256k1: {
+		name: 'secp256k1',
+		byteLength: 32
+	},
+	secp224r1: {
+		name: 'p224',
+		byteLength: 28
+	},
+	prime256v1: {
+		name: 'p256',
+		byteLength: 32
+	},
+	prime192v1: {
+		name: 'p192',
+		byteLength: 24
+	},
+	ed25519: {
+		name: 'ed25519',
+		byteLength: 32
+	}
+};
+aliases.p224 = aliases.secp224r1;
+aliases.p256 = aliases.secp256r1 = aliases.prime256v1;
+aliases.p192 = aliases.secp192r1 = aliases.prime192v1;
+function ECDH(curve) {
+	if (!(this instanceof ECDH)) {
```

Comment:
Can we do this [consistently](https://github.com/crypto-browserify/createHmac/pull/1/files#diff-de1a5a7de991855714cdf433e9f8be96R9) across modules?